packages/dashboard/src/features/analytics/components/AnalyticsConfigDialog.tsx

```
@@ -0,0 +1,127 @@
+import { DisconnectDialog } from './posthog/DisconnectDialog';
+
+const ANALYTICS_SETUP_PROMPT =
+  "I'm using InsForge as my backend platform. I want to add product analytics to this project. Read the current directory and use the InsForge skill to set up PostHog analytics for me.";
+
+interface AnalyticsConfigDialogProps {
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+  connection: PosthogConnection;
+}
+
+export function AnalyticsConfigDialog({ open, onOpenChange, connection }: AnalyticsConfigDialogProps) {
```

ESLint

[error] Replace `·open,·onOpenChange,·connection·` with `⏎··open,⏎··onOpenChange,⏎··connection,⏎`

(prettier/prettier)